include/Architecture.hpp

```diff
@@ -53,6 +53,7 @@ class Architecture {
 	static constexpr bool VERBOSE = false;
 
 public:
+    std::vector<std::pair<short, short>> teleportation_qubits{};
```

Comment:
Very minor: shouldn't this be named `teleportationQubits` in order to conform with (most of) the other names?
This also holds for the identically named member in the `MappingSettings` class.